pennylane/tape/unwrap.py

```diff
@@ -93,6 +105,9 @@ class UnwrapTape:
 

```

Comment:
This `UnwrapTape` class looks very similar to the `Unwrap` class, is there a reason for this repitition? It seems to be a special case of the more general `Unwrap` class?

Reply:
To reduce code complexity! `UnwrapTape` applies to a single tape, `Unwrap` accepts _multiple_ tapes (and simply calls `UnwrapTape` as needed)